doc/iterations/2025.1/update_table_description_from_representation.adoc

```diff
@@ -0,0 +1,79 @@
+= (M) Shape
+
+== Problem
+
+On a table representation, we want to be able to modify the specification used as description of a table and then reload the table with this new specification.
+
+== Key Result
+
+Enable user to modify the description used by a table representation and update existing representation with the new description.
+
+== Solution 1 - Force update all the representation when creating a fork of the description
```

Comment:
I think that we will only consider this first solution, you can remove the other one, we won't consider something like that. It will be as simple as possible to implement it quickly.